helm/prometheus-rules/templates/kaas/turtles/alerting-rules/capi-kubeadmconfig.rules.yml

```diff
@@ -0,0 +1,26 @@
+{{- if eq .Values.managementCluster.provider.flavor "capi" }}
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels: {{- include "labels.common" . | nindent 4}}
+  name: capi-kubeadmconfig.rules
+  namespace: {{.Values.namespace}}
+spec:
+  groups:
+    - name: capi-kubeadmconfig
+      rules:
+        - alert: KubeadmConfigDataSecretGenerationFailed
+          expr: capi_kubeadmconfig_status_condition{type="DataSecretGenerationFailed"}
+          for: 1h
+          labels:
+            area: kaas
+            cancel_if_monitoring_agent_down: "true"
+            cancel_if_outside_working_hours: "true"
```

Comment:
Do we not want this outside working hours? Wouldn't this prevent nodes from launching?